app/controllers/anonymous_feedback/export_requests_controller.rb

```diff
@@ -26,6 +28,18 @@ def show
 
 private
 
+  def set_path_set_id_param
+    return true if anonymous_feedback_params[:path_set_id].presence
```

Comment:
Do we need to return `true` or can we just return on its own? And should it be `.present?` rather than `presence`.

Reply:
whoops, thanks!